README.md

```diff
@@ -55,23 +55,32 @@ $ docker exec -it consul_consul_3 consul info | grep num_peers
 
 ## Using this in your own composition
 
-The Consul service definition can be dropped into any Docker Compose file. Set the ContainerPilot configuration for each other service to use the `CONSUL` environment variable as its Consul target and populate this with the CNS name. On Triton, you should consider using a Consul agent in the application container as a `coprocess`, and point this agent to the `CONSUL` environment variable. The relevant section of the ContainerPilot configuration might look like this:
+The Consul service definition can be dropped into any Docker Compose file. Set the ContainerPilot configuration for each other service to use the `CONSUL` environment variable as its Consul target and populate this with the CNS name. On Triton, you should consider using a Consul agent within the application container as a `job`, and point this agent to the `CONSUL` environment variable. The relevant section of the ContainerPilot configuration might look like this:
 
-```json
+```json5
 {
-  "consul": "localhost:8500",
-  "coprocesses": [
+  consul: "localhost:8500",
+  jobs: [
     {
-      "command": ["/usr/local/bin/consul", "agent",
-                  "-data-dir=/data",
-                  "-config-dir=/config",
-                  "-rejoin",
-                  "-retry-join", "{{ .CONSUL }}",
-                  "-retry-max", "10",
-                  "-retry-interval", "10s"],
-      "restarts": "unlimited"
-    }]
-  }
+      name: "consul-agent",
+      restarts: "unlimited",
+      exec: [
+        "/usr/bin/consul", "agent",
+          "-data-dir=/data",
```

Comment:
Indentation looks off compared to the previous line.

Reply:
That was deliberate to show the flags under the main command itself.

Reply:
I _think_ the health check on the agent needs to be a bit more robust. If I read this correctly, this will mark the agent as healthy even if it isn't yet in sync on raft which might cause issues. Perhaps something like 
 ```
   if [ $(consul info | awk '/members/{print$3}') == 1 ]; then
        _log "No peers in raft"
        consul join ${CONSUL}
    fi
```
?

Reply:
@jacobloveless I don't have a quick answer for that at the moment but I'll take a look. Let's move this discussion to a separate issue over at #45.